x-pack/legacy/plugins/monitoring/public/directives/main/internal_monitoring.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+  This is a temporary deprecation warning that will be valid for the span of 7.6.0 - 8.0.0
+  Context: https://github.com/elastic/stack-monitoring-dev/issues/76
+*/
+
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { i18n } from '@kbn/i18n';
+import { EuiLink, EuiCallOut, EuiFlexGroup, EuiFlexItem, EuiButtonIcon } from '@elastic/eui';
+import { getPageData } from '../../lib/get_page_data';
+import { npSetup } from 'ui/new_platform';
+import { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } from 'ui/documentation_links';
+
+const { cloud } = npSetup.plugins;
+const isCloudEnabled = Boolean(cloud && cloud.isCloudEnabled);
+let checked = false;
+let isLegacy = false;
+let optOut = false;
+
+const getElement = () => document.getElementById('internal-monitoring-warning');
+
+export const destroyInternalMonitoringWarning = () => {
+  const el = getElement();
+  el && unmountComponentAtNode(el);
+};
+
+const callOutProps = {
+  title: i18n.translate('xpack.monitoring.internalCollectionDetectedTitle', {
+    defaultMessage: 'Internal Collection Detected',
+  }),
+  color: 'warning',
+  iconType: 'help',
```

Comment:
I would remove the icon, I don't think it adds any value. - but that is just my personal opinion.
```suggestion
```